js/initcontext.go

```diff
@@ -151,24 +151,25 @@ func (i *InitContext) Require(arg string) goja.Value {
 	}
 }
 
-type moduleInstanceCoreImpl struct {
+// TODO this likely should just be part of the initialized VU or at least to take stuff directly from it.
+type moduleVUImpl struct {
```

Comment:
Can we just use `moduleVU` instead of `moduleVUImpl`?

Reply:
I have some hopes I will be working on the TODO above this cycle, and it will just get completely removed